module/documents/item.mjs

```diff
@@ -533,6 +532,17 @@ export default class Item5e extends SystemDocumentMixin(Item) {
     return !this.system.attuned && (this.system.attunement === "required");
   }
 
+  /* -------------------------------------------- */
+  /*  Data Initialization                         */
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  clone(...args) {
+    const item = super.clone(...args);
+    if ( !item.parent ) item.prepareFinalAttributes();
+    return item;
+  }
```

Comment:
This doesn't take into account the `save` option, which makes the return value a `Promise`. We probably also need to always call `prepareFinalAttributes`, regardless of whether the Item has a `parent`. Cloned Items are not placed into their parents' collections (unless `save` is `true`), so `prepareFinalAttributes` wouldn't be called by the parent.

```suggestion
  clone(data={}, options={}) {
    if ( options.save ) return super.clone(data, options);
    const item = super.clone(data, options);
    item.prepareFinalAttributes();
    return item;
  }
```

Reply:
If it doesn't have a parent then it calls `prepareFinalAttributes` itself at the end of `prepareData`.